src/test/java/test/asserttests/ArrayEqualityAssertTest.java

```diff
@@ -191,6 +191,28 @@ public void arrayInsideSetAssertEqualsDeep() {
     assertEqualsDeep(set, setCopy, "arrays inside sets are compared by value in assertEqualsDeep");
   }
 
+  @Test
+  public void arrayDeepInSetsAssertEquals() {

```

Comment:
It looks like this test is wrong because the 2 collections are not equal. What do you think?